README.md

```diff
@@ -154,3 +161,28 @@ Functionality is the same as described in https://developer.mozilla.org/en-US/do
 ```js
 const closestElement = closest(targetElement, selectors)
 ```
+
+### @qubit/html
```

Comment:
`@qubit/html` would be a different package right?

Reply:
I was wondering if it should be part of utils because it is directly related to html element creation